requirements-spotify.txt

```diff
@@ -1,3 +1,3 @@
 # Spotify related requirements
  # You need to install these with `sudo pip install --upgrade --force-reinstall -r requirements-spotify.txt`
```

Comment:
sudo pip install --upgrade --force-reinstall -r requirements-spotify.txt
should be pip3